pkg/apicapi/apicapi.go

```diff
@@ -38,6 +38,10 @@ import (
 	"github.com/juju/ratelimit"
 )
 
+// defaultConnectionRefresh is used as connection refresh interval if
+// RefreshInterval is set to 0
+const defaultConnectionRefresh = 80 * time.Second
```

Comment:
Apologies for the delay in reviewing this. The APIC side time out is 80 seconds, but we were earlier setting this to 30 seconds:
https://github.com/noironetworks/aci-containers/commit/fcb6e868f812b6234535454b0a10d5175c362c0d#diff-31ded243a50636001aa9734838222ca0L60

Since we want to refresh before the APIC timeout. Could you please change it to 30?

Reply:
Updated to 30 seconds

Reply:
Thanks @ebati!